hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java

```diff
@@ -4639,6 +4639,10 @@ public FSEditLog getEditLog() {
     return getFSImage().getEditLog();
   }
 
+  public NNStorage getNNStorage() {
```

Comment:
This can be private. This PR is just doing a refactor I think, we don't need the test as well, so can make this private

Reply:
Thanks @ayushtkn for your comments.
Yes, it's about refactoring. I see a lot of places getting NNStorage through FSNamesystem, and I think it makes sense for FSNamesystem to do so.
I will go on working.